### PR TITLE
Change front page to Beaver Barcamp 20

### PR DESCRIPTION
**Pull request by khashf**
- Change front page to Beaver Barcamp 20
- Update event date for beaver Barcamp 20
- Add an "outdated" note for the schedule page

 Per Pam's request

### CODE REVIEW

**Comment by khashf** [comment]
[Update BarCamp website to Barcamp 20](https://trello.com/c/VCxnq0pv/1276-update-barcamp-website-to-barcamp-20)

**Comment by osuosl-manatee** [comment]
This PR has been automatically built and can be viewed here: http://beaver-barcamp-pelican-54.staging.osuosl.org

File: barcamp-theme/templates/index.html [hunk 1]
```diff
@@ -7,10 +7,8 @@ <h2>Share Your Passion</h2>
         <h4>{{ SITENAME }}</h4>
         <span class="tab--tiny">With <a href="pages/faq.html">Lightning Talks!</a></span>
         <span class="tab--tiny">What is <a href="pages/about.html">Barcamp?</a></span>
-        <h5>April 6, 2019 from 9am-6pm<br />Kelley Engineering Center<br>
-        <!-- <p>Registration is free and will be open soon!</p> -->
-        <p>Registration is free -- sign up today!</p>
-        <a href="http://beaverbarcamp19.eventbrite.com" id="register-button">Register</a>&emsp;
+        <h5>April 4, 2020<br />Kelley Engineering Center<br>
```
Comment (ramereth) [inline barcamp-theme/templates/index.html]:
```suggestion
        <h5>April 4, 2020 from 9am-6pm<br />Kelley Engineering Center<br>
```

File: barcamp-theme/templates/index.html [hunk 2]
```diff
@@ -7,10 +7,8 @@ <h2>Share Your Passion</h2>
...
+        <p>Watch this website for registration information in the early Spring</p>
         <a href="https://create.osufoundation.org/project/14126" id="register-button">Donate</a></h5><br><br>
```
Comment (ramereth) [inline barcamp-theme/templates/index.html]: Please comment out this line.

File: content/pages/schedule.rst [hunk 3]
```diff
@@ -2,7 +2,7 @@ Schedule
 ########
 :slug: schedule
 
-(NOTE: each session is either 25 or 50 minutes long – please don’t run over)
+NOTE: Outdated - for reference only - this schedule was Beaver Barcamp 19 in 2019
```
Comment (ramereth) [inline content/pages/schedule.rst]: IMHO I don't think we need to change this. The schedule has been pretty static over the years and we can certainly change it later if we need to. Please revert this.

**Comment by osuosl-manatee** [comment]
This PR has been automatically built and can be viewed here: http://beaver-barcamp-pelican-54.staging.osuosl.org

**Comment by osuosl-manatee** [comment]
This PR has been automatically built and can be viewed here: http://beaver-barcamp-pelican-54.staging.osuosl.org

**Comment by osuosl-manatee** [comment]
This PR has been automatically built and can be viewed here: http://beaver-barcamp-pelican-54.staging.osuosl.org

**Pull request merged**